uber/custom_tags.py

```diff
@@ -130,6 +130,19 @@ def urlencode(s):
 
 
 @JinjaEnv.jinja_filter
+def url_to_link(url=None, text=None, target=None):
+    # Jinja2 has a 'urlize' filter but it depends on links having `.com` or `http` in the name
+    # This works with relative links and allows you to also define link text
+    if not url:
+        return ''
```

Comment:
Yeah, it was just a thought. I don't have a strong preference. No need to change it if this works for you.